src/entity/user/User.java

```diff
@@ -12,4 +14,5 @@ public interface User {
     void setEmail(String email);
     String getPhone();
     void setPhone(String num);
+    List<String> notification(List<String> notif);
```

Comment:
in interfaces we want to define abstract methods for the classes that implement it. so instead of declaring the type here, we would just do it in the CommonUser class. But since we are adding a new field, we would need to add getters and setters for it beacue we want to do encapsulation. So in our case, i think we would only need a addNotifications method for now. so add addnotififcations in user interface, define it in common user, and call addnotififcation in ur adopt use case